Backend and frontend `make build-image` fails with invalid Dockerfile syntax

Trying to build the backend or frontend Docker images fails because the “here”-style syntax used with the Dockerfile RUN command does not work correctly in the most recent available Docker version. For instance, these ["RUN <<EOT ..."](https://github.com/collective/cookiecutter-plone-starter/blob/main/%7B%7B%20cookiecutter.project_slug%20%7D%7D/backend/Dockerfile#L14-L20) lines in the backend Dockerfile fail like this:

```json
make build-image
+ which python3
+ python3 -c 'import sys; print((int(sys.version_info[0]), int(sys.version_info[1])) >= tuple(map(int, '\''3.8'\''.split('\''.'\''))))'
+ docker build . -t kenmanheimer/myr-plone-backend:latest -f Dockerfile --build-arg PLONE_VERSION=6.0.0
Sending build context to Docker daemon  2.483MB
Error response from daemon: dockerfile parse error line 15: unknown instruction: ./BIN/PIP
make: *** [Makefile:166: build-image] Error 1
```

A similar failure happens with similar syntax in the frontend Dockerfile [here](https://github.com/collective/cookiecutter-plone-starter/blob/main/%7B%7B%20cookiecutter.project_slug%20%7D%7D/frontend/Dockerfile.default#L5-L10), [here](https://github.com/collective/cookiecutter-plone-starter/blob/main/%7B%7B%20cookiecutter.project_slug%20%7D%7D/frontend/Dockerfile.default#L17-L20), and [here](https://github.com/collective/cookiecutter-plone-starter/blob/main/%7B%7B%20cookiecutter.project_slug%20%7D%7D/frontend/Dockerfile.default#L29-L35).

Maybe the `here` syntax works on the version of Docker on someone’s system, but it does not work on the most recent version of Docker available for my Ubuntu 20.04 or Ubuntu 22.04 systems, Docker version 20.10.22 build 3a2c30b.

A solution that should work on recent Docker versions is to instead use continuation lines separated by `&&` command list operators, as is done with the LABEL statement elsewhere in both Dockerfiles, and typically with the RUN command in many Docker files.

I will file a pull request next.

--- a/{{ cookiecutter.project_slug }}/backend/Makefile
+++ b/{{ cookiecutter.project_slug }}/backend/Makefile
@@ -163,4 +163,4 @@ debug: instance/etc/zope.ini ## Run debug console
 
 .PHONY: build-image
 build-image:  ## Build Docker Image
-	@docker build . -t $(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile --build-arg PLONE_VERSION=$(PLONE_VERSION)
+	DOCKER_BUILDKIT=1 @docker build  . -t $(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile --build-arg PLONE_VERSION=$(PLONE_VERSION)

--- a/{{ cookiecutter.project_slug }}/backend/Makefile
+++ b/{{ cookiecutter.project_slug }}/backend/Makefile
@@ -163,4 +163,4 @@ debug: instance/etc/zope.ini ## Run debug console
 
 .PHONY: build-image
 build-image:  ## Build Docker Image
-	DOCKER_BUILDKIT=1 @docker build . -t $(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile --build-arg PLONE_VERSION=$(PLONE_VERSION)
+	@DOCKER_BUILDKIT=1 docker build . -t $(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile --build-arg PLONE_VERSION=$(PLONE_VERSION)

--- a/{{ cookiecutter.project_slug }}/backend/Makefile
+++ b/{{ cookiecutter.project_slug }}/backend/Makefile
@@ -163,4 +163,4 @@ debug: instance/etc/zope.ini ## Run debug console
 
 .PHONY: build-image
 build-image:  ## Build Docker Image
-	DOCKER_BUILDKIT=1 @docker build  . -t $(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile --build-arg PLONE_VERSION=$(PLONE_VERSION)
+	DOCKER_BUILDKIT=1 @docker build . -t $(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile --build-arg PLONE_VERSION=$(PLONE_VERSION)

--- a/{{ cookiecutter.project_slug }}/frontend/Makefile.default
+++ b/{{ cookiecutter.project_slug }}/frontend/Makefile.default
@@ -89,7 +89,7 @@ storybook: ## Generate Storybook
 
 .PHONY: build-image
 build-image:  ## Build Docker Image
-	@docker build . -t $(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile
+	DOCKER_BUILDKIT=1 @docker build . -t $(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile
 
 # Acceptance tests
 

--- a/{{ cookiecutter.project_slug }}/frontend/Makefile.default
+++ b/{{ cookiecutter.project_slug }}/frontend/Makefile.default
@@ -89,7 +89,7 @@ storybook: ## Generate Storybook
 
 .PHONY: build-image
 build-image:  ## Build Docker Image
-	DOCKER_BUILDKIT=1 @docker build . -t $(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile
+	@DOCKER_BUILDKIT=1 docker build . -t $(IMAGE_NAME):$(IMAGE_TAG) -f Dockerfile
 
 # Acceptance tests
 

